### PR TITLE
Fix: Update the rules with simpler format

### PR DESCRIPTION
**Pull request by rlmoser99**
## Because
The zap system is not in alignment with our professional expectations.

## This PR
*  Updates the rules on website to updated format

## Issue
Closes https://github.com/TheOdinProject/moderator/issues/58

## Additional Information
We will not want to merge in this PR until we have the Discord rules channel updated & the Rules Screening process when new people join our server.

## Pull Request Requirements
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
  - `Fix` - bug fixes
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] I have verified all tests and linters pass after making these changes.
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If applicable, this PR includes new or updated automated tests

## Screenshots
Screenshot 2023-03-27 at 6 15 22 AM

Screenshot 2023-03-27 at 6 15 28 AM

Screenshot 2023-03-27 at 6 15 39 AM

Screenshot 2023-03-27 at 6 15 57 AM

### CODE REVIEW

File: app/views/static_pages/community_rules.html.erb [hunk 1]
```diff
@@ -13,166 +13,151 @@
...
+    <h3 id="curriculum-content">Ask questions about content or projects in our curriculum</h3>
+    <p>
+      <span class="fa-regular fa-circle-check mr-1"></span> The purpose of this server is to support people when they run into issues doing our curriculum. This allows our volunteers to know the exact scope of your problem and correct any poor advice. In addition, this limited scope helps us identify areas in our curriculum that we need to improve.
+    </p>
+    <p>
+      <span class="fa-solid fa-triangle-exclamation mr-1"></span> Do not ask for advice/help on homework or personal projects, even if the topic is covered by our curriculum. Instead, research other servers/communities that offer general programming help.
+    </p>
 
-    <h3 id="mini-mod-example">Example of Mini-Modding</h3>
+    <h3 id="curriculum-tools">Ask questions about the tools recommend in our curriculum</h3>
```
Comment (CouchofTomato) [inline app/views/static_pages/community_rules.html.erb:67]: > Ask questions about the tools recommend in our curriculum

*recommended*

File: app/views/static_pages/community_rules.html.erb [hunk 2]
```diff
@@ -13,166 +13,151 @@
...
 
+    <h3 id="respect">Treat everyone with respect</h3>
     <p>
-      This is not a comprehensive list of all possible scenarios and zaps are made at the discretion of the moderation team if particular behaviors are inappropriate but not otherwise explicitly covered. Please DM @ModMail if you would like to discuss these rules or any moderation activity (whether you are on the receiving end or not).
+      <span class="fa-regular fa-circle-check mr-1"></span> The culture of this community is very similar to the professional workplace messaging apps of our volunteer team. Learning how to respectfully interact with others from around the world will prepare you to interact with your future teammates.
     </p>
+    <p>
+      <span class="fa-solid fa-triangle-exclamation mr-1"></span> We have zero tolerance for disrespect. Therefore, the following behaviors will result in a ban from our server:
+      <ul>
+        <li>Bigotry, such as racism, homophobia, hate speech, etc.</li>
```
Comment (CouchofTomato) [inline app/views/static_pages/community_rules.html.erb:145]: > Bigotry, such as racism, homophobia, hate speech, etc.

Only this one has a `.`

File: app/views/static_pages/community_rules.html.erb [hunk 3]
```diff
@@ -13,166 +13,151 @@
...
+        <li>NSFW or other highly offensive images in posts, profile pictures, or username</li>
+        <li>Spamming multiple channels with the same or nonsensical messages (copypasta)</li>
+        <li>Disrespectful or targeted harassment, directed insults, naming & shaming</li>
+        <li>Aggressive arguments</li>
+        <li>Inciting drama, provoking/baiting other users, passive aggression</li>
+        <li>Unsolicited self promotion</li>
```
Comment (CouchofTomato) [inline app/views/static_pages/community_rules.html.erb:151]: > Unsolicited self promotion

self-promotion ?

File: app/views/static_pages/community_rules.html.erb [hunk 4]
```diff
@@ -13,166 +13,151 @@
...
+        <li>Piracy, links to pirated material, suggesting pirating intellectual property</li>
+        <li>Plagiarism</li>
+        <li>Doxxing</li>
+        <li>Trolling</li>
+        <li>Publicly arguing about receiving moderation after being asked to discuss privately through ModMail.</li>
```
Comment (CouchofTomato) [inline app/views/static_pages/community_rules.html.erb:156]: > Publicly arguing about receiving moderation after being asked to discuss privately through ModMail.

One of the few with a `.`

File: app/views/static_pages/community_rules.html.erb [hunk 5]
```diff
@@ -13,166 +13,151 @@
...
+        <li>Any other behavior deemed unacceptable and deserving of a ban by the moderation team, such as displaying a pattern of breaking our rules.</li>
```
Comment (CouchofTomato) [inline app/views/static_pages/community_rules.html.erb:157]: > Any other behavior deemed unacceptable and deserving of a ban by the moderation team, such as displaying a pattern of breaking our rules.

One of the few with a `.`

**Comment by rlmoser99** [comment]
@CouchofTomato Thanks again for looking at the new wording with a fine tooth comb. I've made all of the suggested changes.

**Comment by ManonLef** [comment]
This looks excellent Rachel. Thanks for doing this!
I had some notes written out but Austin caught them and then some. 

I am looking at a section where we could add a bit about 'professionalism' and 'memes' perhaps. Maybe someone else has any ideas about this while I go sleep on it.

**Comment by rlmoser99** [comment]
>  I am looking at a section where we could add a bit about 'professionalism' and 'memes' perhaps

@ManonLef I'm all for adding memes to the rules,  but I don't think this initial roll out is the best time to do so. I'd rather create a new issue/discussion about this for us to address in the coming weeks.

**Comment by ManonLef** [comment]
That makes perfect sense and I now also remember you mentioning keeping the rules themselves the same. 

Was removing the word "professionalism" / "professional" intentional? I do find it convenient mod-wise to point people to our expectations surrounding that when chat goes wild. 

But that can also wait. I'm just curious and forgot to mention or ask it initially. We can talk about it when the reworked rules are up if you prefer.  

Can't wait to see these up!

File: app/views/static_pages/community_rules.html.erb [hunk 6]
```diff
@@ -13,166 +13,151 @@
...
+      <li>Help others by guiding them to the solution in a public 1:1 conversation</li>
+      <li>Make it easy for others to help you by asking a detailed question</li>
+      <li>Share resources that are relevant to our curriculum</li>
+      <li>Report misconduct to ModMail</li>
+      <li>Treat everyone with respect</li>
     </ul>
 
-    <h3 id="zap-level-1">1 Zap - Warning for Minor Violations</h3>
-
+    <h3 id="dont-summary"><span class="fa-solid fa-triangle-exclamation mr-1"></span> Don't</h3>
```
Comment (KevinMulhern) [inline app/views/static_pages/community_rules.html.erb:37]: Same as above but using amber for the color instead. Not 100 sold on the color but I think some variation of yellow would be better than red. Red just seemed too hostile when I tried it lol

```suggestion
    <h3 id="do-summary" class="flex items-center"><span class="fa-solid fa-triangle-exclamation mr-3 text-lg text-amber-600"></span> Don't</h3>
```

Outputs:
Screenshot 2023-03-28 at 09 01 54

File: app/views/static_pages/community_rules.html.erb [hunk 7]
```diff
@@ -13,166 +13,151 @@
...
-        <strong>Any other behavior deemed unacceptable and deserving of a ban by the moderation team</strong> such as not correcting <a href="https://slash7.com/2006/12/22/vampires/" target="_blank">“help vampire”</a> behaviors.
-      </li>
-    </ul>
-
-    <h3 id="decay-example">Decay Explanation</h3>
-
+    <h3 id="safe-topics">Discuss topics that are safe and welcoming for everyone</h3>
     <p>
-      Zaps decay at a rate of 1 zap per week. E.g., if you receive 5 zaps on a Friday, the following Friday you will have 4, the Friday after that you will have 3, and so forth. This means higher level infractions remain on record for a longer period of time and jeopardizes your participation in the server if rules continue to be broken.
+      <span class="fa-regular fa-circle-check mr-1"></span> Discuss topics that are relevant to our curriculum or have an off-topic channel.
```
Comment (KevinMulhern) [inline app/views/static_pages/community_rules.html.erb:53]: The smaller icons don't need as much, just a little extra spacing and the same colors as the title icons:
```suggestion
      <span class="fa-regular fa-circle-check mr-2 text-green-700"></span> Discuss topics that are relevant to our curriculum or have an off-topic channel.
```

Outputs:
Screenshot 2023-03-28 at 09 03 08

**Review by KevinMulhern — COMMENTED** [review]
Just a few styling/design suggestions, nice work with this 👏

**Comment by rlmoser99** [comment]
> Was removing the word "professionalism" / "professional" intentional? I do find it convenient mod-wise to point people to our expectations surrounding that when chat goes wild.

@ManonLef Great question. Let's work out the details and discuss this on this PR:
https://github.com/TheOdinProject/moderator/pull/62

**Pull request merged**